component/steps/ApplicationsSteps.scala

```diff
@@ -79,8 +83,10 @@ class ApplicationsSteps extends ScalaDsl with EN with Matchers with NavigationSu
   }
 
   Given("""^I have no application assigned to my email '(.*)'$""") { (email: String) =>
-    ApplicationStub.configureUserApplications(email)
-    ApplicationStub.configureUserApplications(email)
+    val userId = idOf(email)
+
+    ApplicationStub.configureUserApplications(userId)
+    ApplicationStub.configureUserApplications(userId)
```

Comment:
Do we know why this is here twice ?

Reply:
No idea but it seems that we don't need both of them :)